app/views/shared/_status.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="<%= dialog_column_class %>">
+  <h1>Status of your request</h1>
+  <hr />
+  <dl class='dl-horizontal dl-invert'>
+    <dt>Requested on</dt>
+    <dd><%= request.created_at.to_formatted_s(:long_ordinal) %></dd>
```

Comment:
I don't know what `long_ordinal` is, but could this be e.g. `<%= l request.created_at %>`? [1]


[1] http://guides.rubyonrails.org/i18n.html#adding-date-time-formats